Library/Homebrew/cmd/style.rb

```diff
@@ -1,16 +1,12 @@
-#:  * `style` [`--fix`] [`--display-cop-names`] [<formulae>|<files>]:
+#:  * `style` [`--fix`] [`--display-cop-names`] [<formulae>|<files>|<taps>]:
```

Comment:
I think we can but given it's not a departure, as you mention, I think it's fine as-is. The ordering here reflects the code priority.